server/api/auth.go

```diff
@@ -185,30 +185,30 @@ func (a *API) handleRegister(w http.ResponseWriter, r *http.Request) {
 	//     schema:
 	//       "$ref": "#/definitions/ErrorResponse"
 	if a.MattermostAuth {
-		a.errorResponse(w, r.URL.Path, http.StatusNotImplemented, "not permitted in plugin mode", nil)
+		a.customErrorResponse(w, r.URL.Path, http.StatusNotImplemented, "not permitted in plugin mode", nil)
```

Comment:
Probably a new `NotImplemented` error would be good here.

Reply:
Addressed in [1753287](https://github.com/mattermost/focalboard/pull/3792/commits/1753287d742a6de4013d85a194b0fb9fdb2decc4), only one `customErrorResponse` remains for `request entity too large`